src/components/BucketItems.js

```diff
@@ -50,14 +53,11 @@ class BucketItems extends Component {
                 this.setState({
                     items: response.data.item
                 })
-           }).catch((error)=>{
-               console.log(error)
-            })
+           }).catch((error)=>{})
```

Comment:
What are you doing with the error here? Why don't you either put it in an object in state and maybe present it to the user. The `.catch` here feels misplaced.